src/modules/DataSave.js

```diff
@@ -0,0 +1,15 @@
+// Fetch/GET data/TODOs from the localStorage
+const getFromLocalStorage = () => {
+  let data = JSON.parse(localStorage.getItem('todos'));
+  if (!data) {
+    data = [];
+  }
+  return data;
+};
+
+// Saves data/ TODOs in the localStorage
+const saveToLocalStorage = (todos) => {
+  localStorage.setItem('todos', JSON.stringify(todos));
+};
+
```

Comment:
Good job here but there is a way to make it shorter
`const saveToLocalStorage = (todos) => localStorage.setItem('todos',JSON.stringify(todos));`

Reply:
Thanks @OnlyUnkown for your review I will definitely fix them.